DeploymentExperiences.md

```diff
@@ -1,8 +1,37 @@
+Link to my Django App [here](https://github.com/Nicocchi/heroku-deploy-test)
+
+Link to the Heroku server
+
+Logging in [here](https://django-test-heroku.herokuapp.com/admin)
```

Comment:
This is great, you might also consider eventually adding a registration page to enable a new user to sign up